uaclient/api/u/unattended_upgrades/status/v1.py

```diff
@@ -42,18 +43,22 @@ class UnattendedUpgradesStatusResult(DataObject, AdditionalInfo):
     fields = [
         Field("systemd_apt_timer_enabled", BoolDataValue),
         Field("apt_periodic_job_enabled", BoolDataValue),
-        Field("package_lists_refresh_frequency_days", IntDataValue),
+        Field(
+            "package_lists_refresh_frequency_days",
+            IntDataValue,
```

Comment:
tiniest of nits: looks like the new comma caused this to be black-formatted differently